### PR TITLE
std.string.center_unittest_example

### PR DESCRIPTION
**Pull request by burner**
* added unittest doc examples for center

### CODE REVIEW

File: std/string.d [hunk 1]
```diff
@@ -3875,7 +3875,15 @@ S center(S)(S s, size_t width, dchar fillChar = ' ')
...
-@trusted pure
+///
+@safe pure unittest
+{
+    assert(center("hello", 7, 'X') == "XhelloX");
+    assert(center("hello", 2, 'X') == "hello");
+    assert(center("hello", 9, 'X') == "XXhelloXX");
+}
+
+@safe pure
```
Comment (JackStouffer) [inline std/string.d]: Any particular reason to have these on separate lines?

Comment (burner) [inline std/string.d]: because I only changed the line.

**Comment by atilaneves** [comment]
LGTM

**Comment by JackStouffer** [comment]
LGTM

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

**Pull request merged**